website/src/react-native/showcase.js

```diff
@@ -868,7 +868,14 @@ var apps = [
     icon: 'https://s3-ap-northeast-1.amazonaws.com/sw-misc/sharewis3_app.png',
     link: 'https://itunes.apple.com/jp/app/id585517208',
     author: 'ShareWis Inc.'
-  }
+  },
+	{
```

Comment:
I know I am being boring here, but did you notice that the indent level is not the same as few lines above? ;/ 
<img width="86" alt="screen shot 2016-04-25 at 11 52 10" src="https://cloud.githubusercontent.com/assets/2464966/14780209/2abdeb9e-0adc-11e6-89f4-2142b49b734f.png">